### DISCUSSION TITLE
Microsoft.AspNetCore.SpaServices version 5.0.0 not available?

### DISCUSSION THREAD

**Discussion by appimpact** [dotnet/aspnetcore#28459] [Q&A] [answered]
Hello,

We have noticed a strange discrepancy between other AspNetCore packages and [Microsoft.AspNetCore.SpaServices](https://www.nuget.org/packages/Microsoft.AspNetCore.SpaServices) package.

All packages have been bumped to 5.0.0 but this one is still at 3.1.10.

Even [Microsoft.AspNetCore.SpaServices.Extensions](https://www.nuget.org/packages/Microsoft.AspNetCore.SpaServices.Extensions) package is at 5.0.0.

Is this intentional or someone forgot to bump it?

Thanks

**Comment by Pilchie** [comment, marked as answer]
That package was obsoleted in .NET Core 3.1, and is not included in .NET 5.  See https://github.com/aspnet/Announcements/issues/379.
